web/src/js/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import * as modalAction from '../../ducks/ui/modal'
+import ModalList from './ModalList'
+
+class PureModal extends Component {
+
+    constructor(props, context) {
+        super(props, context)
+    }
+
+    render() {
+        const { activeModal, hideModal } = this.props
+        const ActiveModal = _.find(ModalList, m => m.name === activeModal )
```

Comment:
Haven't seen your Slack discussion, but my five cents: If `activeModal` is not undefined, I think it's fine to assume that `.find()` is sucessful, so there shouldn't be a need for a fallback. It's ok (if not useful) if it breaks visibly if there's an invalid state.

Other than that, no need to use lodash here - ES6 has Array.prototype.find. 😃